src/editor/Editor.js

```diff
@@ -2401,6 +2401,11 @@ define(function (require, exports, module) {
         }
     };
 
+    // CDO-Bramble: Set readOnly mode.
+    Editor.prototype.setReadonly = function(value) {
```

Comment:
tiny nit: can you be consistent about `readonly` vs `readOnly`? No preference for which one but it would be good to have everything the same.

Reply:
yeah, good call. i'll stick with `readOnly` everywhere since that matches the CodeMirror setting